### ISSUE TITLE
Improve text readability with transparency

### ISSUE THREAD

**Issue by nick87720z** [QtCurve/qtcurve#19]
The most popular way is to reduce transparency just behind text. The only question is how better to do it. There could be two ways for that:
simple (or coarse) - Create single blurred cloud (also could be some logarithmic gradient) for entire label in alpha channel;
blurred text - according to name, blurred text copy in alpha channel. Somewhat close to that - outline in cairo-dock's dialogs and tooltips (may be disabled).

**Comment by yuyichao** [comment]
Yep, that's on the plan and thank you for bring it up (although not with highest priority).
I still need to figure out if it is possible to do. It probably is given that Bespin is also doing sth special to the text renderring.

**Comment by nick87720z** [comment]
Honestly did not know about bespin. But i have seen it in plasma. Also same wishes were submited for oxygen.
